### PR TITLE
Screen/cart

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added cart page

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Good to merge

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://xendesign.netlify.app/js/index.js"></script>
+    <title>ZenCart</title>
+</head>
+
+<body class="bg-dark  overflow-hidden text-light">
+    <header
+        class=" bg-dark-2 l-2 r-2 t-2 p-2 pl-3 pr-3 h-auto fixed flex items-center z-50 br-lg b-solid b-1 h-min br-primary back-blur-5 bx-sh-primary-3">
+        <nav class="flex items-center justify-between w-full ">
+            <ul class="left row items-center gap-2 justify-between">
```
Comment (Ankur9669) [inline cart.html]: Question: I am not sure if you shoud use nav tag inside header

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+                <div class="card card-horizontal card-dark">
+                    <div class="card-header">
+
+                        <img src="images/behringer-headphone.png" alt="BEYERDYNAMIC" class="object-contain">
+                    </div>
+                    <div class="card-bottom">
+
+                        <div class="card-body ">
+                            <div class="card-title">BEYERDYNAMIC - DT 770 PRO</div>
+                            <p class="card-description">
```
Comment (Ankur9669) [inline cart.html:77]: fix: card-title should be p tag

File: css/style.css [hunk 3]
```diff
@@ -1,11 +1,270 @@
...
+  margin-left: auto;
+  visibility: hidden;
+}
+nav ul.left,
+nav ul.right {
+  flex-wrap: nowrap;
+}
+@media (max-width: 535px) {
+  header nav ul.left .input-box {
+    display: none;
```
Comment (Ankur9669) [inline css/style.css:26]: fix: This is very high specificity, you should reduce it for sure
also use @media only screen and max-width

File: css/style.css [hunk 4]
```diff
@@ -1,11 +1,270 @@
...
+  }
+  header nav ul.left {
+    width: 100%;
+    display: flex;
+    justify-content: space-between !important;
+  }
```
Comment (Ankur9669) [inline css/style.css:32]: fix: remove !important it is very bad and also reduce specificity

File: css/style.css [hunk 5]
```diff
@@ -1,11 +1,270 @@
...
 .section-category .img {
+  transition: all 0.1s linear;
   background-color: var(--dark-color-7);
   object-fit: contain;
 }
+.hero-section-button {
+  padding: 0.8rem 1rem;
+}
+li a.btn {
+  font-size: 0.9rem;
```
Comment (Ankur9669) [inline css/style.css]: Suggestion: There must be a better way for doing this

File: css/style.css [hunk 6]
```diff
@@ -1,11 +1,270 @@
...
+}
+form.form {
+  background-color: var(--light-grey);
+  margin-bottom: 15vh;
+  margin-top: 0vh;
+}
+@media (max-width: 535px) {
+  form.form {
+    padding: 1rem;
+  }
```
Comment (Ankur9669) [inline css/style.css:102]: Same

File: css/style.css [hunk 7]
```diff
@@ -1,11 +1,270 @@
...
+  width: 1em;
+  border-radius: 50em;
+  background: url(https://pro.fontawesome.com/releases/v5.10.0/svgs/solid/times-circle.svg)
+    no-repeat 50% 50%;
+  background-size: contain;
+  opacity: 0;
+  pointer-events: none;
+  filter: invert(1);
+}
+
```
Comment (Ankur9669) [inline css/style.css:116]: fix: I am not sure if this is required

File: css/style.css [hunk 8]
```diff
@@ -1,11 +1,270 @@
...
+.sidebar .col {
+  padding: 0.5rem;
+}
+section.products {
+  flex: 12;
+  /* padding: 1rem;
+  padding-top: 0;
+  overflow: hidden;
+  overflow-y: scroll; */
+  height: 100vh;
```
Comment (Ankur9669) [inline css/style.css:144]: Question: why flex: 12??
fix: also remove commented code

File: css/style.css [hunk 9]
```diff
@@ -1,11 +1,270 @@
...
+    left: 0;
+    box-shadow: var(--bx-sh-primary-1);
+  }
+  .filter-btn {
+    display: block;
+  }
+}
+.show {
+  display: flex !important;
+}
```
Comment (Ankur9669) [inline css/style.css:181]: fix: remove !important

File: css/style.css [hunk 10]
```diff
@@ -1,11 +1,270 @@
...
+  height: 100%;
+  padding: 1rem;
+  /* width: 100%; */
+  object-fit: contain;
+}
+.card-horizontal .card-bottom {
+  max-width: max-content;
+
+  flex: auto;
+}
```
Comment (Ankur9669) [inline css/style.css:209]: Suggestion: I think by default flex is auto only

File: css/style.css [hunk 11]
```diff
@@ -1,11 +1,270 @@
...
+  justify-content: center;
+  height: 100vh;
+  overflow: hidden;
+  overflow-y: scroll;
+}
+
+.cart-products {
+  width: 35rem;
+  margin-bottom: 30vh;
+  overflow: hidden;
```
Comment (Ankur9669) [inline css/style.css:246]: Ouestion: I am not sure why margin-bottom is 30vh

File: index.html [hunk 12]
```diff
@@ -13,44 +13,53 @@
 
 <body class="bg-dark  overflow-x-hidden text-light">
     <header
-        class="nav bg-dark-2 l-2 r-2 t-2  p-2 pl-3 pr-3 h-auto fixed row items-center z-50 br-lg b-solid b-1 h-min br-primary back-blur-5 bx-sh-primary-3">
-        <nav class="row items-center justify-between w-full ">
-            <ul class="left row items-center">
-                <h1 class="text-lg text-light">ZenCart</h1>
+        class=" bg-dark-2 l-2 r-2 t-2 p-2 pl-3 pr-3 h-auto fixed flex items-center z-50 br-lg b-solid b-1 h-min br-primary back-blur-5 bx-sh-primary-3">
+        <nav class="flex items-center justify-between w-full ">
```
Comment (Ankur9669) [inline index.html]: Question: Why such a big class name

File: login.html [hunk 13]
```diff
@@ -13,58 +13,69 @@
 
 <body class="bg-dark  overflow-x-hidden text-light">
     <header
-        class="nav bg-dark-2 l-2 r-2 t-2  p-2 pl-3 pr-3 h-auto fixed row items-center z-50 br-lg b-solid b-1 h-min br-primary back-blur-5 bx-sh-primary-3">
-        <nav class="row items-center justify-between w-full ">
-            <ul class="left row items-center">
-                <h1 class="text-lg text-light">ZenCart</h1>
+        class=" bg-dark-2 l-2 r-2 t-2 p-2 pl-3 pr-3 h-auto fixed flex items-center z-50 br-lg b-solid b-1 h-min br-primary back-blur-5 bx-sh-primary-3">
+        <nav class="flex items-center justify-between w-full ">
```
Comment (Ankur9669) [inline login.html]: Suggestion: I am not sure about class names

File: css/style.css [hunk 14]
```diff
@@ -1,11 +1,270 @@
...
+  }
+  .card-horizontal .card-header {
+    max-width: 100%;
+    height: min-content;
+  }
+
+  .card-horizontal .card-bottom {
+    max-width: max-content;
+
+    height: min-content;
```
Comment (Ankur9669) [inline css/style.css:233]: fix: remove unused spacing

File: css/style.css [hunk 15]
```diff
@@ -1,11 +1,270 @@
...
+  overflow-y: scroll;
+  padding-left: 2%;
+  padding-bottom: 25vh;
+}
+.filter-btn {
+  display: none;
+}
+@media (max-width: 950px) {
+  section.products {
+    width: 100vw;
```
Comment (Ankur9669) [inline css/style.css:155]: Suggestion: you can use products only

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
The UI looks great!!

But i am not sure about the naming conventions and the css specificity that you have used, you have used highly specific selectors which i think you should get rid of

File: index.html (hunk 12 above)
Comment (sanjitsarkar) [reply index.html]: It's a utility class

**Review by bossmonkey98 — APPROVED** [review]

File: css/style.css (hunk 3 above)
Comment (sanjitsarkar) [reply css/style.css:26]: Done fixing

**Pull request merged**